.github/.jira_sync_config.yaml

```diff
@@ -0,0 +1,39 @@
+settings:
+    # Jira project key to create the issue in
+  jira_project_key: "LXD"
+  
+  # Dictionary mapping GitHub issue status to Jira issue status
+  status_mapping:
+    opened: Untriaged
+    closed: done 
+    
+  # (Optional) Jira project components that should be attached to the created issue
+  # Component names are case-sensitive
+  # components:
+  #  - LXD
+    - MicroCloud
```

Comment:
Should this line also be commented out?

Reply:
In MicroCluster it's commented out: https://github.com/canonical/microcluster/blob/main/.github/.jira_sync_config.yaml.

Or uncomment the `components:` line above.

Reply:
I'll comment out, no value having it in the configuration